etcdserver/membership/cluster.go

```diff
@@ -307,21 +307,21 @@ func (c *RaftCluster) RemoveMember(id types.ID) {
 	c.removed[id] = true
 }
 
-func (c *RaftCluster) UpdateAttributes(id types.ID, attr Attributes) bool {
+func (c *RaftCluster) UpdateAttributes(id types.ID, attr Attributes) {
```

Comment:
the pervious code was confusing. 

Basically it should be a switch on path. When it goes in https://github.com/coreos/etcd/pull/5005/files#diff-971f6af906b31cc86efd71aff56aa96bR1094 here, it should update the store. The previous code was designed to save a few lines of code, so it reuse the store operation here: https://github.com/coreos/etcd/pull/5005/files#diff-971f6af906b31cc86efd71aff56aa96bR1106.

Now we move the update logic to: https://github.com/coreos/etcd/pull/5005/files#diff-befbd92c479c58c96793f5e6fa7ea851R103. So membership pkg owns the membership path of the store.